src/plugins/default/ErrorPlugin.ts

```diff
@@ -20,6 +37,14 @@ export class ErrorPlugin implements IEventPlugin {
 
         let result = parser.parse(context, exception);
         if (!!result) {
+          let additionalData = this.getAdditionalData(exception);
+          if (!!additionalData) {
```

Comment:
If additional data is an empty object it shouldn't be added to extra properties as we will render empty extra exception properties if the key exists.